src/crates.md

```diff
@@ -46,10 +46,13 @@ which, together with the seed, can be used to reconstruct the generator's state.
 ## WASM support
 
 Almost all Rand crates support WASM out of the box. Only the `rand_core` crate [may
-require enabling features for WASM support](https://github.com/rust-random/getrandom#features).
+require enabling features for WASM support](https://github.com/rust-random/getrandom#features),
+mainly for seeding entropy via the OS-provided APIs.
 Consequently, if you are using another crate depending on `rand_core` (such as
-most Rand crates), you may have to explicitly enable `getrandom` features for it
-to work.
+most Rand crates), you may have to explicitly enable `getrandom` features for seeding
+to work. Alternatively, in case you are developing for a sandboxed or unknown
+WASM platform (`wasm32-unknown-unknown`) that can't depend on environment provided
+APIs, you might want to disable `getrandom` and seed the generator manually.
```

Comment:
I don't like the wording here much: it's the `getrandom` crate which doesn't support `wasm32-unknown-unknown`, and `rand_core` (thus also `rand`) are dependants of that.

Additionally, I'm not sure everyone will follow how to disable `getrandom`?

> you may have to explicitly enable `getrandom` features

Also somewhat unclear — just link to the getrandom doc.

Reply:
Ahh, actually I thought it's a bit janky too, but I wasn't too sure about changing the wording too much as I'm not too well-versed in `rand`. I'll rephrase it a bit to my best knowledge!

Reply:
Rephrased it!